kevm-pyk/src/kevm_pyk/summarizer.py

```diff
@@ -607,6 +607,110 @@ def create_kcfg_explore() -> KCFGExplore:
                 f.write('\n')
         return passed
 
+    def _transform_underscores(self, inner: KInner) -> KInner:
```

Comment:
None of these methods refer to `self`, so they could be `@staticmethod`-s, or preferably, even module-level functions.